client/src/css/Alternative.css

```diff
@@ -1,43 +1,45 @@
 .Alternative {
-  margin: 1rem 0;
-  background: #6b71a5;
-  color: #ffffff;
-  font-size: 1.8rem;
+  margin: 0.5rem 0;
+  background-color: #6B71A5;
+  color: #FFF;
   display: flex;
-  box-shadow: 0px -2px 0px 0px rgba(0, 0, 0, 0.25) inset,
-              0px 2px 1px 0px rgba(0, 0, 0, 0.2);
+  box-shadow: 0 -0.125rem 0 0 rgba(0, 0, 0, 0.25) inset,
+              0 0.125rem 0.0625rem 0 rgba(0, 0, 0, 0.2);
 }
 
 .Alternative:last-of-type {
   margin-bottom: 0;
 }
 
 .Alternative:hover {
-  background: #8d93c7;
+  background-color: #8D93C7;
```

Comment:
I was going off some article that I skimmed through last night, it all made sense at the time, but seeing as I can't find it anymore I'm just assuming I had a hallucination or something. [This](https://github.com/mdo/css-perf#background-vs-background-color) actually indicates that background is the preferred method anyways.

Will switch to lowercase, doesn't matter at all to me, just made an arbitrary choice at the time. PyCharm will convert all of them in an instant.